src/utils/getTscErrors.ts

```diff
@@ -166,6 +166,7 @@ async function getProjectErrors(projectPath: string, tsRepoPath: string, stdout:
         if (error.text) {
             // TODO: use replaceAll once that's available.
             error.text = error.text.split(tsRepoPath).join("<ts>");
+            error.fileUrl = error.fileUrl?.split(tsRepoPath).join("<ts>");
```

Comment:
use `?.` for both?

Reply:
You mean `.text`? That one's always set.

Reply:
Oh I'm stupid, this is in a block. I just noticed.